.github/workflows/inspektor-gadget.yml

```diff
@@ -54,6 +54,8 @@ jobs:
       with:
         go-version: ${{ env.GO_VERSION }}
       id: go
+    - name: Install debian packages
+      uses: ./.github/actions/install-debian-packages
```

Comment:
I was the one who removed `Install debian packages` from the Lint step. I removed it and nothing broke back then 😅 
A comment would be good here

Reply:
We should set it up so that the cache gets invalidated when we change something in `./github/workflows`